runtime/doc/api.txt

```diff
@@ -48,7 +48,7 @@ version.api_compatible	API is backwards-compatible with this level
 version.api_prerelease	Declares the current API level as unstable >
 			(version.api_prerelease && fn.since == version.api_level)
 functions		API function signatures
-ui_events		UI event signatures |rpc-remote-ui|
+ui_events		UI event signatures |remote-ui|
```

Comment:
`|ui|` is sufficient. Just like we claimed `|api|`, we can make "UI" a single idea when communicating with users. Details about "msgpack" and "remote" don't need to be called out in most contexts.

Reply:
Agree completely